docs/DeveloperGuide.adoc

```diff
@@ -836,6 +836,58 @@ Use case ends.
 Use case resumes at step 2.
 
 [discrete]
+=== Use case: Undo last command
+
+*MSS*
+
+1. User requests to undo last command
+2. AddressBook confirms if user really wishes to undo last command
```

Comment:
Can we confirm that this behaviour is what we want? We will have to extend the current undo/redo functionality to prompt for confirmation in this case.

Reply:
@Xenonym I guess it is decided then - better not to have such reversible actions as per the Prof's recommendation.